src/Native/Runtime/amd64/InteropThunksHelpers.S

```diff
@@ -4,19 +4,43 @@
 
 .intel_syntax noprefix
 #include <unixasmmacros.inc>
-
-//TODO: Implement these stub method for non-windows
+#include <AsmOffsets.inc>         // generated by the build from AsmOffsets.cpp
+#define POINTER_SIZE 8
 
 LEAF_ENTRY RhpCommonStub, _TEXT
-    int 3
+
+    PUSH_ARGUMENT_REGISTERS
+
+    // +8 for alignment
+    alloc_stack    (SIZEOF_MAX_FP_ARG_SPILL + 8)
```

Comment:
`src\System.Private.TypeLoader\src\Internal\Runtime\TypeLoader\ReferenceSource\amd64\asmconstants.h` exists just for reference. It is not actually used by the CoreRT builds. (However, there is work to do on the TypeLoader to make it work on x64 Unix.)

@shrah added it in this change.